### PR TITLE
Frontendforgotpasswordcomponent

### PR DESCRIPTION
**Pull request by folusho-adeyemi**
## Description

Adding the frontend components for the forgot-password feature
....I WILL HAVE TO UPDATE THE UI AND ALSO MAKE USE OF EXTERNAL API TO FETCH REAL DATA

## Milestones
Milestone 2

## Resources
[<links to tutorials, code snippets, inspirations>](https://dev.to/jlong4223/how-to-implement-email-functionality-with-node-js-react-js-nodemailer-and-oauth2-2h7m)- This article really explained how to go about the  Gmail OAuth2 configuration Process
<if you took or translated specific code from the internet, please call it out here!>

## Test Plan
<div>
    <a href="https://www.loom.com/share/88c0f7e4cd324aeb9ed8af13ba3647cc">
      <p>FashionConnect - 21 July 2023 - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/88c0f7e4cd324aeb9ed8af13ba3647cc">
      
    </a>
  </div>

##VIDEO WALKTHROUGH
Link to video: https://www.loom.com/share/88c0f7e4cd324aeb9ed8af13ba3647cc?sid=3511e0f4-0610-4c25-a1bd-ecc551f08271

### CODE REVIEW

File: FashionConnect_api/email.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import nodemailer from "nodemailer";
```
Comment (safirshahbaaz) [inline FashionConnect_api/email.js:1]: I think all this code is already in your previous PR. Can you update this PR for only the frontend changes?

**Review by safirshahbaaz — COMMENTED** [review]
Looks like these changes were already made in the previous PR. Can you update this PR to only the frontend changes?

File: FashionConnect_ui/src/components/PasswordChange/PasswordChange.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        } catch (error) {
+            // Handle any network or API request errors
+            alert('Login failed: ' + error);
+        }
+    };
+
+    return (
+        <div className="signup-form-container">
+            <form className="signup-form" onSubmit={handleSubmit}>
+                <h2>Sign Up</h2>
```
Comment (safirshahbaaz) [inline FashionConnect_ui/src/components/PasswordChange/PasswordChange.jsx:46]: This shouldn't be Sign up right? I think something more like Change Password.

**Review by safirshahbaaz — APPROVED** [review]

File: FashionConnect_ui/src/components/PasswordChange/PasswordChange.jsx (hunk 2 above)
Comment (folusho-adeyemi) [reply FashionConnect_ui/src/components/PasswordChange/PasswordChange.jsx:46]: yeah!
I mistakenly overlooked this.

**Pull request merged**